src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Win32.cs

```diff
@@ -64,6 +64,8 @@ private void StartRaisingEvents()
         /// <summary>Stop monitoring the current directory.</summary>
         private void StopRaisingEvents()
         {
+            _enabled = false;
```

Comment:
Not really feedback, more just curiosity around what this helps: what does moving the _enabled=false call up here give us? :)